static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import * as segmentUtil from 'common/modules/experiments/segment-util';
+import * as testCanRunChecks
+    from 'common/modules/experiments/test-can-run-checks';
+import * as viewLog from 'common/modules/commercial/acquisitions-view-log';
+import alwaysAsk
+    from 'common/modules/experiments/tests/contributions-epic-always-ask-strategy';
+import askFourEarning
+    from 'common/modules/experiments/tests/contributions-epic-ask-four-earning';
+import acquisitionsEpicLiveBlog
+    from 'common/modules/experiments/tests/acquisitions-epic-liveblog';
+import acquisitionsEpicTestimonialsRoundTwo
+    from 'common/modules/experiments/tests/acquisitions-epic-testimonials-round-two';
+import acquisitionsEpicSingleCta
+    from 'common/modules/experiments/tests/acquisitions-epic-single-cta';
+
+/**
+ * acquisition tests in priority order (highest to lowest)
+ */
+const tests = [
+    alwaysAsk,
+    acquisitionsEpicSingleCta,
+    acquisitionsEpicTestimonialsRoundTwo,
+    askFourEarning,
+    acquisitionsEpicLiveBlog,
+];
+
+export const epicEngagementBannerTests = tests.reduce((out, Test) => {
+    const testInstance = new Test();
+
+    if (testInstance.isEngagementBannerTest) {
+        out.push(testInstance);
+    }
+    return out;
+}, []);
+
+export const abTestClashData = tests.map(Test => new Test());
+
+export const getTest = () => {
```

Comment:
`ABTest` ultimately, but not until all of the tests are converted. I'll add a comment